admin/includes/modules/copy_product_confirm.php

```diff
@@ -47,52 +41,56 @@
                     SET products_quantity_order_units = 1
                     WHERE products_id = " . $products_id;
             $results = $db->Execute($sql);
+            $product->fields['products_quantity_order_units'] = 1; 
         }
         // fix Product copy from if Minimum is 0
         if ($product->fields['products_quantity_order_min'] == 0) {
             $sql = "UPDATE " . TABLE_PRODUCTS . "
                     SET products_quantity_order_min = 1
                     WHERE products_id = " . $products_id;
             $results = $db->Execute($sql);
+            $product->fields['products_quantity_order_min'] = 1; 
         }
 
-        $tmp_value = zen_db_input($product->fields['products_quantity']);
-        $products_quantity = (!zen_not_null($tmp_value) || $tmp_value == '' || $tmp_value == 0) ? 0 : $tmp_value;
-        $tmp_value = zen_db_input($product->fields['products_price']);
-        $products_price = (!zen_not_null($tmp_value) || $tmp_value == '' || $tmp_value == 0) ? 0 : $tmp_value;
-        $tmp_value = zen_db_input($product->fields['products_weight']);
-        $products_weight = (!zen_not_null($tmp_value) || $tmp_value == '' || $tmp_value == 0) ? 0 : $tmp_value;
-
-        $db->Execute("INSERT INTO " . TABLE_PRODUCTS . " (products_type, products_quantity, products_model, products_image,
-                                                      products_price, products_virtual, products_date_added, products_date_available,
-                                                      products_weight, products_status, products_tax_class_id,
-                                                      manufacturers_id, products_quantity_order_min, products_quantity_order_units,
-                                                      products_priced_by_attribute, product_is_free, product_is_call, products_quantity_mixed,
-                                                      product_is_always_free_shipping, products_qty_box_status, products_quantity_order_max,
-                                                      products_sort_order, products_price_sorter, master_categories_id)
-                  VALUES ('" . zen_db_input($product->fields['products_type']) . "',
-                          '" . $products_quantity . "',
-                          '" . zen_db_input($product->fields['products_model']) . "',
-                          '" . zen_db_input($product->fields['products_image']) . "',
-                          '" . $products_price . "',
-                          '" . zen_db_input($product->fields['products_virtual']) . "',
-                          now(),
-                          " . (zen_not_null(zen_db_input($product->fields['products_date_available'])) ? "'" . zen_db_input($product->fields['products_date_available']) . "'" : 'null') . ",
-                          '" . $products_weight . "', '0',
```

Comment:
The `0` here refers to `products_status`, which we hard-code to 0 on copy.
But I don't see that accommodated in this PR. Was that intentional?

Reply:
- Nope - accidental miss.